docs/appendices/release-notes/5.9.2.rst

```diff
@@ -47,4 +47,5 @@ See the :ref:`version_5.9.0` release notes for a full list of changes in the
 Fixes
 =====
 
-None
+- Fixed a BWC issue resulting in an exception when doing a rolling upgrade of a
+  cluster with custom users.
```

Comment:
```suggestion
- Fixed a BWC issue resulting in an exception when using custom user/roles created on a CrateDB version < :ref:`version_5.9.0`.
```

Reply:
Thanks for the suggestion. Applied and adjusted to fit in 80 symbols per line.